packages/dtx-desktop/src/main/simfile-service.ts

```
@@ -263,3 +263,188 @@ export async function createSimfileRecord(
+						label = baseFileName.toUpperCase();
+						console.log(`Using filename-based label for ${fileName}: ${label}`);
+					}
+
+					parsedLevels.push({
+						label: label,
+						level: dtx.level
+					});
+				}
+			} catch (error) {
+				console.warn(`Failed to parse DTX file ${fileName}:`, error);
+				continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)